### PR TITLE
wording fix

### PR DESCRIPTION
**Pull request by psteinb**

### CODE REVIEW

File: source/lesson05/content.rst [hunk 1]
```diff
@@ -63,4 +65,4 @@ Exercises
   y = df_iris['species'].values
 
 
-* (optional) We built an extremely simple feed-forward network model. To experiment more, load the [MNIST database of handwritten digits](http://keras.io/datasets/) and see if you can beat a standard scikit-learn classifier. Unlike the Iris dataset, this is a situation where the power and relative complexity of neural networks is justified. Try it yourself first, but if you get stuck, take a look at [this notebook](https://github.com/wxs/keras-mnist-tutorial/blob/master/MNIST%20in%20Keras.ipynb).
+* (optional) We built an extremely simple feed-forward network model. To experiment more, load the [MNIST database of handwritten digits](http://keras.io/data sets/) and see if you can beat a standard scikit-learn classifier. Unlike the Iris data set, this is a situation where the power and relative complexity of neural networks is justified. Try it yourself first, but if you get stuck, take a look at [this notebook](https://github.com/wxs/keras-mnist-tutorial/blob/master/MNIST%20in%20Keras.ipynb).
```
Comment (Vindaar) [inline source/lesson05/content.rst:68]: > http://keras.io/data sets/

`s/datasets/data sets/` was a bit too much here ;)

File: source/lesson08/content.rst [hunk 2]
```diff
@@ -43,7 +44,7 @@ Check your Learning
 Exercises
 =========
 
-* through `OpenML <https://docs.openml.org/Datasets/>`_ or `keras datasets <https://keras.io/api/datasets/>`_
+* through `OpenML <https://docs.openml.org/Datasets/>`_ or `keras data sets <https://keras.io/api/data sets/>`_
```
Comment (Vindaar) [inline source/lesson08/content.rst:47]: Same as above (data sets in URL).

File: source/lesson05/content.rst (hunk 1 above)
Comment (psteinb) [reply source/lesson05/content.rst:68]: thanks, I submitted fixes immediately, see #17.

File: source/lesson08/content.rst (hunk 2 above)
Comment (psteinb) [reply source/lesson08/content.rst:47]: thanks for spotting this.

**Pull request merged**